Gradata/docs/contributing/deprecation-policy.md

```
@@ -0,0 +1,37 @@
+# Deprecation Policy
+
+## Scope
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)